boards/arm/bl654_dvk/bl654_dvk.yaml

```diff
@@ -12,3 +12,4 @@ supported:
   - ble
   - pwm
   - watchdog
+vendor: nordic
```

Comment:
lairdconnect

Reply:
then you will have to fix the DTS 

https://github.com/zephyrproject-rtos/zephyr/blob/2bc980f83190b22f4aa3f6b1560b40ece649ed95/boards/arm/bl652_dvk/bl652_dvk.dts#L14